src/lib/polymer-bootstrap.html

```diff
@@ -37,13 +37,28 @@
       var options = {
         prototype: prototype
       };
-      // NOTE: we're specifically supporting older Chrome versions here 
+      // NOTE: we're specifically supporting older Chrome versions here
       // (specifically Chrome 39) that throw when options.extends is undefined.
       if (prototype.extends) {
         options.extends = prototype.extends;
       }
-      Polymer.telemetry._registrate(prototype);
-      document.registerElement(prototype.is, options);
+
+      try {
```

Comment:
There's two more bugs left to nail down for try/catch to no longer cause deopts in V8. Getting super close!